pkg/syncer/util.go

```diff
@@ -196,3 +202,16 @@ func migratedToAnnotationExists(ctx context.Context, annotations map[string]stri
 	log.Debugf("%v annotation not found", common.AnnMigratedTo)
 	return false
 }
+
+// GetSCNameFromPVC gets name of the storage class from provided PVC
```

Comment:
Can you add Unit test for this function?

Reply:
done :)